usagov/usagov-2021
# Usagov 1409 reduce height homepage

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1409

## Description

Reduce the bottom margin of ul.how-button-list from 16px to 0px.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



1. Go to the homepage (localhost)
2. Check that the “How do I” section looks like this:
<img width="1068" alt="image" src="https://github.com/usagov/usagov-2021/assets/39020809/b0fe5748-06ea-41ff-b09c-0a9343f680ef">


### Requires New Config

### Requires New Content

### Validation Steps

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
4. Find the commit of this pull request.
5. Build and deploy the changes.
6. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
